blob/fileblob/fileblob_test.go

```diff
@@ -35,17 +35,23 @@ import (
 
 type harness struct {
 	dir       string
+	prefix    string
 	server    *httptest.Server
 	urlSigner URLSigner
 	closer    func()
 }
 
-func newHarness(ctx context.Context, t *testing.T) (drivertest.Harness, error) {
+func newHarness(ctx context.Context, t *testing.T, prefix string) (drivertest.Harness, error) {
```

Comment:
I test the new functionality in `fileblob_test` and `memblob_test`; I think that should be sufficient, no?

`fileblob_test` tests the URL part too (`memblob_test` just checks that it doesn't fail).

Reply:
SGTM